File: Go/Relatorio.go
package listaRicardo

import "fmt"

type Relatorio struct {
	Conteudo string
}

// Método que implementa MostrarTexto para Relatorio
func (r Relatorio) MostrarTexto() {
	fmt.Printf("Conteúdo do relatório:\n%s\n", r.Conteudo)
}

File: Java/Relatorio.java
public class Relatorio implements Interface_Texto {
    private String conteudo;

    public Relatorio(String conteudo){
        this.conteudo = conteudo;
    }

    @Override
    public void mostrarTexto() {
        System.out.println("Conteúdo do relatório: \n"+this.conteudo);
    }
}
